src/services/import-service.spec.js

```diff
@@ -311,4 +311,100 @@ describe('import-service', () => {
 
   });
 
+  describe('getImportBatches', () => {
+    let result;
+
+    beforeEach(async () => {
+      result = await sut.createImportBatch({ importSource, documentsToImport, user });
+      await taskStore.updateOne({ 'taskParams.key': 'v5NSMktadhzquVUAEdzAKg' }, { $set: { processed: true } });
+    });
+
+    it('should return the batch', async () => {
+      result = await sut.getImportBatches();
+
+      expect(result).toEqual([
+        {
+          _id: expect.stringMatching(/\w+/),
+          createdBy: user._id,
+          createdOn: expect.any(Date),
+          completedOn: null,
+          batchType: BATCH_TYPE.importDocuments,
+          batchParams: {
+            name: 'source-1',
+            hostName: 'source1.com',
+            allowUnsecure: false
+          },
+          errors: [],
+          progress: 0.5
+        }
+      ]);
+    });
+  });
+
+  describe('getImportBatch', () => {
```

Comment:
'getImportBatchDetails'

Reply:
Fixed, thanks, I used the wrong method name at first, then forgot to update the title of the describe :).